CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 <!-- There is always Unreleased section on the top. Subsections (Added, Changed, Fixed, Removed) should be added as needed. -->
 
 ## Unreleased
+### Fixed
+- Check for equality of user ids in `UserMerge` - so that we don't "merge" user into the same user id effectively deleting it.
```

Comment:
Vincent said that deleting behaviour will be fixed. So maybe it is not necessary to mention it in docs. Hmm?